src/test_workflow/dependency_installer_opensearch.py

```diff
@@ -37,6 +37,6 @@ def install_build_dependencies(self, dependency_dict: dict, dest: str) -> None:
         """
         os.makedirs(dest, exist_ok=True)
         for dependency, version in dependency_dict.items():
-            path = f"{self.root_url}/builds/opensearch/plugins/{dependency}-{version}.zip"
+            path = os.path.join(self.root_url, f"builds/opensearch/plugins/{dependency}-{version}.zip")
```

Comment:
I think you want to make this Windows compatible too here, I believe `os.path.join(self.root_url, 'builds', 'opensearch', ...)` should work.